tests/integration/test_logs.py

```diff
@@ -188,6 +188,7 @@ def test_put_subscription_filter_lambda(
                     else None
                 ),
                 replacement="event_id",
+                replace_reference=False,
```

Comment:
Had to disable reference replacement, as on CircleCI the id ended up being "200" and replaced the HTTP response code.
In that case the "event_id" isn't used as reference anywhere so this is fine. 
Still we should keep this in mind for other cases as the reference replacement may have unwanted side effects. /cc @dominikschubert

Reply:
Yeah integer values like that will be a PITA but I think having the event id here replaced globally doesn't add that much benefit anyway.